src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java

```diff
@@ -83,11 +84,29 @@ public void initGui() {
                 GRAY + "inaccessible locations."
         ), (v) -> showInaccessibleLocations, (i, btn) -> showInaccessibleLocations = !showInaccessibleLocations);
 
-        addButton(MapButtonType.PIN, 2, Arrays.asList(
+        addButton(MapButtonType.PIN, 1, Arrays.asList(
                 LIGHT_PURPLE + "[>] Show Seaskipper routes",
                 GRAY + "Click here to enable/disable",
                 GRAY + "Seaskipper routes."
         ), (v) -> showSeaskipperRoutes, (i, btn) -> showSeaskipperRoutes = !showSeaskipperRoutes);
+
+        addButton(MapButtonType.BOAT, 2, Arrays.asList(
+                LIGHT_PURPLE + "[>] Buy a boat",
+                GRAY + "Click here",
+                GRAY + "to buy a boat.",
+                GRAY + "Only works if you don't have one."
```

Comment:
```suggestion
                LIGHT_PURPLE + "[>] Buy a boat",
                GRAY + "Click here to buy a boat."
```

Also, a good idea would be to disable the button if the player already has a boat and add the lore with a space.